README.md

```diff
@@ -413,8 +413,26 @@ lines in the `~/.emacs.d/init.el` file:
 ```
 
 ## Spacemacs logo
-For Ubuntu users, follow this guide to
-[change the logo in Unity][cpaulik-unity-icon].
+
+### Ubuntu users
```

Comment:
This section needs to be improved.
You can well describe it in a coherent paragraph.
What I see now are a bunch of broken sentences.
Whenever writing anything, you'd better read it.